### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by Zuwairanajma**
**In this branch i implemented the following**
- Created a new app. Give it a meaningful name.
- Initialized my project with Git.
- Ensured that your project has Postgres database set up.
- Created and ran the necessary migration files as shown in the ERD diagram.
- photo for users table is a link to an image
- Foreign keys were be included.
 - In posts table the column holding the foreign key to the users table was named author_id
- All columns that are foreign keys  have a corresponding index.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.1].define(version: 2023_10_31_101633) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.text "text"
+    t.bigint "author_id", null: false
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The schema diagram shared with you requires that this should be `user_id`, not `author_id`. Kindly rename this table column using a new migration.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id"], name: "index_comments_on_author_id"
+    t.index ["post_id"], name: "index_comments_on_post_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "author_id", null: false
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] The schema diagram shared with you requires that this should be `user_id`, not `author_id`. Kindly rename this table column using a new migration.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Zuwairanajma,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @Zuwairanajma 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented schema in accordance to the given template ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**